packages/docusaurus-theme-bootstrap/src/theme/DocTagsListPage/index.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import type {Props} from '@theme/BlogTagsListPage';
+
+function DocTagsListPage(props: Props): JSX.Element {
```

Comment:
The bootstrap theme should also be able to render this comp, otherwise the deploy preview won't pass.
We need to implement it in both themes